src/vivarium_csu_ltbi/tools/ltbi_results_preprocessing.py

```diff
@@ -0,0 +1,89 @@
+import pandas as pd
+
+result_dir = '/ihme/costeffectiveness/results/vivarium_csu_ltbi/'
+
+path_for_location = {
+    'ethiopia_end_100': result_dir + 'updated-input-data-end-100/ethiopia/2020_01_02_16_41_59',
+    'india_end_100': result_dir + 'updated-input-data-end-100/india/2020_01_02_16_36_19',
+    'peru_end_100': result_dir + 'updated-input-data-end-100/peru/2020_01_02_16_36_14',
+    'philippines_end_100': result_dir + 'updated-input-data-end-100/philippines/2020_01_02_16_36_14',
+    'south_africa_end_100': result_dir + 'updated-input-data-end-100/south_africa/2020_01_02_16_36_27',
+    'ethiopia_end_10': result_dir + 'updated-input-data-end-10/ethiopia/2020_01_02_17_00_21',
+    'india_end_10': result_dir + 'updated-input-data-end-10/india/2020_01_02_17_00_23',
+    'peru_end_10': result_dir + 'updated-input-data-end-10/peru/2020_01_02_17_01_17',
+    'philippines_end_10': result_dir + 'updated-input-data-end-10/philippines/2020_01_02_17_01_17',
+    'south_africa_end_10': result_dir + 'updated-input-data-end-10/south_africa/2020_01_02_17_01_26'
+}
+
+output_dir = '/home/j/Project/simulation_science/latent_tuberculosis_infection/result/sim_raw_hdf/'
+
+def load_data(country_path):
+    df = pd.read_hdf(country_path + '/output.hdf')
+    df = df.drop(columns='random_seed').reset_index()
+    df.rename(columns={'configuration.ltbi_treatment_scale_up.scenario': 'scenario'},
+              inplace=True)
+    
+    scenario_count = (df
+                      .groupby(['input_draw', 'random_seed'])
+                      .scenario
+                      .count())
+    idx_completed = scenario_count[scenario_count == 3].reset_index()
+    idx_completed = idx_completed[['input_draw', 'random_seed']]
+    df_completed = pd.merge(df,
+                            idx_completed,
+                            how='inner',
+                            on=['input_draw', 'random_seed'])

```

Comment:
This seems more complicated than necessary---what is the goal, precisely?

Reply:
In case we have unfinished jobs and want to generate reasonable results, here I selected unique input draw and random seed pair with all 3 scenarios finished. It becomes redundant if no failure jobs.

Reply:
ok, I see why this inner join is convenient instead of a call to `pd.Series.isin` or something more straight-foward in this case---you are selecting based on two columns.